tooling/template-base/components/CTA/ButtonLink.tsx

```diff
@@ -1,13 +1,25 @@
 import React from 'react';
-import logo from '../../renderer/share.svg';
+import share from '../Assets/share';
+
+const element = (
```

Comment:
This actually being used? Think it's just a duplicate of `share`, so can just remove it.